syntax = "proto3";

package capsule;

import "capsule/common.proto";

service Scheduler {
    rpc SaveUrl (Url) returns (Empty);
    rpc ListSnapshots (Url) returns (SnapshotList);

    rpc RegisterWorker (Endpoint) returns (Empty);
    rpc RegisterStorage (Endpoint) returns (Empty);
}
